raft/src/main/java/org/apache/kafka/raft/internals/VoterSet.java

```diff
@@ -95,6 +95,21 @@ public Optional<Node> voterNode(int voterId, ListenerName listenerName) {
             .map(address -> new Node(voterId, address.getHostString(), address.getPort()));
     }
 
+    /**
+     * Return true the provided voter node is a voter and would cause a change in the voter set.
```

Comment:
nit: `Return true if the provided voter node ...`

Reply:
Done.